example/javascriptlib/basic/6-client-server-realistic/build.mill

```diff
@@ -0,0 +1,68 @@
+package build
+import mill._, javascriptlib._
+
+object client extends ReactScriptsModule
+
+object server extends JestModule {
+  def npmDeps =
+    Seq("@types/cors@^2.8.17", "@types/express@^5.0.0", "cors@^2.8.5", "express@^4.21.1")
+
+  def npmDevDeps =
+    super.npmDevDeps() ++ Seq("@types/supertest@^6.0.2", "supertest@^7.0.0")
+
+  override def testConfigSource = Task.Source(millSourcePath / "jest.config.ts")
+
+  override def bundleFlags = Map("external" -> Seq("express", "cors"))
+
+  override def mkENV = Task {
+    super.mkENV() ++ Map("PORT" -> "3001")
+  }
+
+}
+
+object app extends RsWithExpressModule {
+  def clientMod = client
+  def serverMod = server
+}
+
+// Documentation for mill.example.javascriptlib
```

Comment:
If you got your todomvc example from somewhere on the internet, you should include a link to it

Reply:
Its the react version of the one in scalalib